lib/tasks/deployment/20210507200644_fix_transition_aged_youth_for_nil_birth_month_year_youth.rake

```diff
@@ -0,0 +1,16 @@
+namespace :after_party do
+  desc "Deployment task: fix_transition_aged_youth_for_nil_birth_month_year_youth"
+  task fix_transition_aged_youth_for_nil_birth_month_year_youth: :environment do
+    puts "Running deploy task 'fix_transition_aged_youth_for_nil_birth_month_year_youth'"
```

Comment:
Do after_party tasks run on prod? Should I add an `unless Rails.env.production?`

Reply:
Yes, they do. Please do.